### ISSUE TITLE
Can't dup Symbol 

### ISSUE THREAD

**Issue by 97-109-107** [sup-heliotrope/sup#477]
Upon pressing '/' sup crashes with the following

````
[2015-09-21 17:25:39 +0200] ERROR: oh crap, an exception
--- TypeError from thread: main
can't dup Symbol
/home/xxx/.gem/ruby/2.2.0/gems/highline-1.7.6/lib/highline/question.rb:34:in `dup'
/home/xxx/.gem/ruby/2.2.0/gems/highline-1.7.6/lib/highline/question.rb:34:in `initialize'
/home/xxx/.gem/ruby/2.2.0/gems/highline-1.7.6/lib/highline.rb:252:in `new'
/home/xxx/.gem/ruby/2.2.0/gems/highline-1.7.6/lib/highline.rb:252:in `ask'
/home/xxx/Opt/sup/lib/sup/modes/scroll_mode.rb:82:in `search_in_buffer'
/home/xxx/Opt/sup/lib/sup/mode.rb:59:in `handle_input'
/home/xxx/Opt/sup/lib/sup/buffer.rb:222:in `handle_input'
/home/xxx/Opt/sup/bin/sup:297:in `<module:Redwood>'
/home/xxx/Opt/sup/bin/sup:116:in `<main>'
````

I'm running sup from git directly, I've just re-indexed my mail with ``sup-sync``. I've tried my luck with both highline 1.7.6 and 1.6.21 - both result in the same error.

**Comment by abinoam** [comment]
Dear sup developers,

We are releasing a bug fix that makes HighLine::Question to coerce its "question" (question template string) to String first. So, if you send it a Symbol it will be correctly translated into a String first. This makes the Question interface more flexible.

But, could you please check if at your code you are really willing to send a Symbol to HighLine::Question#new  (through HighLine#ask)?

PR: https://github.com/JEG2/highline/pull/160

Thank you in advance and Best regards

**Comment by gauteh** [comment]
Belive this is fixed in #478
